src/main/java/edu/cornell/kfs/gl/CuGeneralLedgerConstants.java

```diff
@@ -5,6 +5,7 @@
 public class CuGeneralLedgerConstants extends GeneralLedgerConstants {
 
 	public static final String ANNUAL_CLOSING_CHARTS_PARAM = "ANNUAL_CLOSING_CHARTS";
+	public static final String COLLECTOR_HEADER_RECORD_TYPE = "HD";
```

Comment:
Since this a constant for a hard coded value used by the Collector related code in the GL module as well I think it makes sense to keep it in this file.

Initially I thought this was a new value used only by the Concur related code hence the suggestion to move it in the Concur module.